core/templates/pages/contributor-dashboard-page/opportunities-list-item/opportunities-list-item.component.ts

```diff
@@ -80,7 +98,8 @@ export class OpportunitiesListItemComponent {
     if (this.opportunity) {
       if (this.opportunity.progressPercentage) {
         this.progressPercentage = (
-          this.opportunity.progressPercentage + '%');
+          String((Math.floor(this.opportunity.progressPercentage))) + '%'
```

Comment:
Could we use a template literal for this to avoid the String cast?

Reply:
Done